docker-compose.yml

```diff
@@ -51,11 +66,21 @@ services:
     volumes:
       - ./db/data:/var/lib/postgresql/data
 
-  postgres_admin:
-    image: dpage/pgadmin4:latest
+  # postgres_admin:
+  #   image: dpage/pgadmin4:latest
+  #   depends_on:
+  #     - postgres
+  #   ports:
+  #     - 15432:80
+  #   env_file:
+  #     - .env
+
+  adminer:
+    image: adminer:latest
     depends_on:
       - postgres
     ports:
-      - 15432:80
+      - 15432:8080
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Verify Adminer port change intention</b></div>
<div id="fix">

Consider verifying if changing the port from `80` to `8080` for the Adminer service is intentional and aligns with the default Adminer port. This change might affect how users access the Adminer interface.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -83 +83 @@
 -      - 15432:8080
 +      - 15432:80
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/omkardongre/messenger-api/pull/6#issuecomment-2451737561>#148eb1</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged